src/Phinx/Wrapper/TextWrapper.php

```diff
@@ -41,18 +41,23 @@
 class TextWrapper
 {
     private $app;
-    private $env;
+    private $options;
     private $exit_code;
 
     /**
-     * @param  Phinx\Console\PhinxApplication $app
-     * @param  String                         $default_env  environment fallback, defaults to "development"
-     * @return void
+     * @param PhinxApplication $app
+     * @param array $options
      */
-    public function __construct(PhinxApplication $app, $default_env = 'development')
+    public function __construct(PhinxApplication $app, Array $options = array())
     {
+        $options += array(
+            'environment' => 'development',
+            'config_path' => './phinx.yml',
```

Comment:
It would be better if this option name matched the command line name "configuration".